docs/intro/glossary.md

```diff
@@ -0,0 +1,241 @@
+
+In the SDK it is normally used as a synonym for [solar panel](#solar-panel).
+
+### SoC
+
+[State of charge](#state-of-charge). See the [Wikipedia article](https://en.wikipedia.org/wiki/State_of_charge) for more details.
+
+### SoP
+
+[State of power](#state-of-power).
+
+## Microgrid
+
+A local electrical grid that connects a set of different [types of components](#component-category) together. It can be connected to the public [grid](#grid), or be completely isolated, in which case it is known as an island.
+
+### Component Category
+
+The category [components](#component) of a [microgrid](#microgrid) belong to.
+
+[Components](#component) of the same category have the same characteristics (for example offer the same set of [metrics](#metric)).
+
+#### Battery
+
+A storage system for electrical energy.
+
+#### CHP Plant
+
+A generator that produces combined heat and power ([CHP](#chp)). Usually powered via combustion of some form of fuel.
+
+#### Converter
+
+A [DC](#dc)-DC converter.
+
+#### EV Charger
+
+A station for charging [EVs](#ev).
+
+#### Electrolizer
+
+An device that converts water into hydrogen and oxygen.
+
+#### Grid
+
+A point where the local [microgrid](#microgrid) is connected to the public electricity grid.
+
+#### Inverter
+
+A device or circuitry that converts [DC](#dc) electricity to [AC](#ac) electricity.
+
+#### Meter
+
+A device for measuring electrical [metrics](#metrics) (for example current, voltage, etc.).
+
+#### PV Array
+
+A collection of [PV](#pv) panels.
+
+#### Pre-charge module
+
+A device that gradually ramp the [DC](#dc) voltage up to prevent any potential damage to sensitive electrical components, like capacitors.
+
+While many [inverters](#inverter) and [batteries](#battery) come equipped with in-built pre-charging mechanisms, some may lack this feature. In such cases, external pre-charging modules can be used.
+
+#### Relay
+
+A device that generally have two states: open (connected) and closed (disconnected).
+
+They are generally placed in front of another [component](#component), e.g., an [inverter](#inverter), to control whether the component is connected to the [microgrid](#microgrid) or not.
+
+#### Sensor
+
+A device for [measuring](#measurement] ambient [metrics](#metric) (for example temperature, humidity, etc.).
+
+#### Solar Panel
+
+A panel with [PV](#pv) cells that generates [DC](#dc) electricity from sunlight.
+
+#### Wind Turbine
+
+A device that converts the wind's kinetic energy into electrical energy.
+
+### Component
+
+A device (of a particular [category](#component-category)) within a [microgrid](#microgrid).
+
+### Component ID
+
+A numeric identifier uniquely representing an instance of a [component](#component) in a particular[microgrid](#microgrid). It is always of type `int`.
+
+For example, a battery an have a component ID **5**.
+
+### Island
+
+A [microgrid](#microgrid) that is not connected to the public electricity [grid](#grid).
```

Comment:
The definition of `consumer` is missing

Reply:
It wasn't in the wiki, I would also leave it for a separate PR.

Reply:
It is, 4th bullet. https://github.com/frequenz-floss/frequenz-sdk-python/wiki/Glossary#microgrid-power-terminology

Reply:
That's `consumer_power`, a terminology thing, IMHO glossary-wise it is covered by the terms **Power** and **Consumption**. Please create an follow-up PR (or an issue including the exact term and definition you want to include) if you think something else should be added.

Reply:
I don't think it should be omitted in the glossary so added it also here https://github.com/frequenz-floss/frequenz-sdk-python/issues/650

Reply:
Great, thanks!